### PR TITLE
Style room (header + input)

### PR DESCRIPTION
**Pull request by JolieClavicule**

### CODE REVIEW

File: style/room.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+html { 
+    background: url('../img/sky.webp') no-repeat center center fixed; 
+    -webkit-background-size: cover;
```
Comment (mathis47000) [inline style/room.css]: remove webkit

File: style/room.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+header {
+    display: flex;
+    justify-content: flex-end;
+    z-index: 1;
+    position: relative;
+}
+
+.title {
+    position: absolute;
```
Comment (mathis47000) [inline style/room.css]: better with flex ?

File: style/room.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+.message-container {
+    flex: 1;
+    overflow-y: auto;
+    padding: 10px;
+}
+
+.input-container {
+    background: url('../img/ground.jpg');
+    background-repeat: repeat-x;
+    background-size: auto;
```
Comment (mathis47000) [inline style/room.css]: duplication

**Review by mathis47000 — COMMENTED** [review]
send message with enter

**Review by flobike47 — APPROVED** [review]

**Pull request merged**